.clang-format

```diff
@@ -0,0 +1,3 @@
+---
+Language: Cpp
+BasedOnStyle: Google
```

Comment:
![image](https://user-images.githubusercontent.com/18620025/94351223-91362700-0024-11eb-841b-280d24e0d11a.png)

What would be best for embedded C++ @PeterGhimself ?

Reply:
can we add new formats?

Reply:
We can create our own set of rules using the 1187729484 different options